files/preStudies/scrum.tex

```diff
@@ -0,0 +1,41 @@
+\section{Scrum tools}
+This pre-study aimed on finding a useful tool for managing the agile software development work flow carried out through out the project.
+
+Here follows a list of features that were, more or less, requirements. Further more it should preferably be free to use for Open Source Software.
+
+\subsection*{Must haves}
+\begin{itemize}
+\item A Scrum board to organize tasks in categories, e.g. \textit{Ready}, \textit{In progress} and \textit{Done}.
+\item A way to manage a backlog of tasks to do.
+\end{itemize}
+
+\subsection*{Nice to haves}
+\begin{itemize}
+\item Automatic integration handling task progress with source control tools, i.e. \textit{GitHub}.
+\item A way to visualize the history of how tasks got processed.
+\end{itemize}
+
+In a general search of solutions the products \textit{Trello} from Trello, Inc., \textit{ZenHub} from Launch Labs, Inc. and \textit{Waffle} from CA Technologies was spotted and further analyzed.
+
+Here follows a brief description of each product.
+
+\subsection*{Trello}
+A simple To Do list web application that enables users to arrange \textit{cards} i.e. tasks in columns of the users desires and as such fulfills the \textit{must have} requirements.
+\textit{Cards} can be assigned labels, check lists, members and has its own discussion board associated.
+
+\subsection*{ZenHub}
+A fully featured project management tool to handle to agile process around software development. 
```

Comment:
to handle to  -> "to handle the agile process"   OR "to handle agile processes " ??